kube-shell/Dockerfile

```diff
@@ -2,34 +2,28 @@ ARG ALPINE=alpine:latest
 FROM ${ALPINE} as alpine
 ARG ARCH
 
+ARG KUBECTL_VERSION=v1.29.0
+ARG HELM_VERSION=v3.13.3
+
 RUN apk add -U --no-cache bash bash-completion curl jq
 
```

Comment:
how come the same changes are showing up after this got merged https://github.com/portainer/docker-images/pull/14/files

Reply:
?

Reply:
woops.  Rebased.